lib/publiccloud/azure.pm

```diff
@@ -505,48 +506,51 @@ sub terraform_apply {
     $args{vars}->{sku} = $sku if ($sku);
 
     my @instances = $self->SUPER::terraform_apply(%args);
-    $self->upload_boot_diagnostics('resource_group' => $self->get_resource_group_from_terraform_show());
+
+    my $instance_id = $self->get_terraform_output('.instance_id.value[0]');
```

Comment:
what I often need is external IP , can we also get it as record_info please ?

Reply:
That will be second commit. It's on my TODO.

Reply:
ok